themes/classic/templates/checkout/_partials/steps/payment.tpl

```diff
@@ -71,7 +71,7 @@
           {if $option.form}
             {$option.form nofilter}
           {else}
-            <form id="payment-form" method="POST" action="{$option.action nofilter}">
+            <form id="payment-{$option.id}-form" method="POST" action="{$option.action nofilter}">
```

Comment:
it could be considered a breaking change, but it's also bad to have several times the same id in our HTML...

what do you think @NeOMakinG?

Reply:
It's an acceptable BC